orbit/cmd/orbit/orbit.go

```diff
@@ -460,6 +565,42 @@ func loadOrGenerateToken(rootDir string) (string, error) {
 	}
 }
 
+func killProcessByName(name string) error {
+	foundProcess, err := getProcessByName(name)
+	if err != nil {
+		return fmt.Errorf("get process: %w", err)
+	}
+	if err := foundProcess.Kill(); err != nil {
+		return fmt.Errorf("kill process %d: %w", foundProcess.Pid, err)
+	}
+	return nil
+}
+
+var errProcessNotFound = errors.New("process not found")
+
+func getProcessByName(name string) (*gopsutil_process.Process, error) {
```

Comment:
I wonder if we'll want to refactor these into some other package for general use at a later time. Not necessary now IMO.